product_brand/tests/test_product_brand.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2018 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestProductBrand(common.TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.product = self.env.ref('product.product_product_4')
+        self.supplier = self.ref('base.res_partner_2')
```

Comment:
@ThomasBinsfeld Here `self.env.ref` not `self.ref`

Reply:
@nikul-serpentcs `self.ref` is a good way to get the id of the record and `self.env.ref` a good way to get the record itself. I won't change it since we only need the ID in this unit test. Please feel free to submit a refactoring PR after the migration merge.

Reply:
@ThomasBinsfeld In case, In future use supplier record then change the `self.env.ref`.
meanwhile, We change this(self.env.ref) and useing `self.supplier.id`.

It's not blocking issue but  It's Good for us.

Reply:
@ThomasBinsfeld If you're review is non-blocking, you can update your review and add an approval. Thanks for the review.